app/assets/javascripts/listings/templates/listing/_additional_sidebar_info.html.slim

```diff
@@ -0,0 +1,24 @@
+.content-group ng-if="::$ctrl.parent.isOwnership($ctrl.parent.listing)"
+  h4.h-caps-lined
+    | Expected move-in date
+  p.t-small.caps
+    | {{ ::$ctrl.parent.listing.Expected_Move_in_Date | date : 'MMMM yyyy' }}
+
+.content-group
+  ng-include src="'listings/templates/listing/_what_to_expect.html'"
+
+.content-group ng-repeat="lotteryPreference in ::$ctrl.parent.listing.preferences" ng-if="!$ctrl.listingHasLotteryResults() && lotteryPreference.pdfUrl"
```

Comment:
I'm still not sure what this section is. @chrisdolendo do you know what it is? I can't find anything like it in the pattern lib too. I'm tempted to get rid of it.